controller/filehandlingcontroller.php

```diff
@@ -109,7 +109,12 @@ public function load($dir, $filename) {
 			}
 
 		} catch (\Exception $e) {
-			return new DataResponse(['message' => 'An internal server error occurred.'], Http::STATUS_BAD_REQUEST);
+			if(method_exists($e, 'getHint')) {
```

Comment:
I'd appreciate if we instead do multiple catch blocks one for the HintException especially.